README.md

```diff
@@ -0,0 +1,87 @@
+# OWL
+
+OWL is a community driven testing lab comprising of an extensible set of open source vendor neutral validation tools and testing frameworks designed to validate the functionality, scalability, and interoperability of wallets, agents, and other components within the decentralized trust ecosystems. 
+
+The OWL name is a nod to the Telecom industry's interoperability "Labs", such as at [UNH](https://www.iol.unh.edu/), where environments are created to enable vendors and product creators to bring their products to test with others. The not-an-acronym name OWL is to prevent confusion with the OpenWallet Foundation's "Labs" maturity level.
+
+## OWL Agent Test Harness
+Agent Test Harness (OATH) is a collaborative platform and environment that is crucial for ensuring that different decentralized trust agent implementations, which are responsible for handling decentralized identifiers (DIDs) and exchanging verifiable credentials, can interoperate seamlessly. OATH currently includes support for Credo-TS and ACA-Py, among others. It's strength is in the emphisis of:
+* Interoperability over individual agent functionality
+* Real agent communication over test or reference agents
+* Protocol RFCs over specific vendor agent implementations
+* Elevating communication and ease of test writing to the level of code
+* Industry-standard test libraries using Behavior-Driven Development (BDD)
+* An API to drive agent interoperability, agnostic of underlying technology.
+
+A summary of OATH's daily interoperability test runs are published on the website [https://aries-interop.info](https://aries-interop.info).
+
+### Features
```

Comment:
I’d be tempted to make this a link to the Features list in the Readme for the respective projects vs. maintaining two copies.  OK if you would prefer it this way — just a suggestion.

Reply:
Yes, that is a good idea. I'll merge it like it is and make a note to make sure the test harness repos have updated feature lists and then link them here instead.